src/Microsoft.DotNet.Wpf/redist/D3DCompiler/D3DCompiler.vcxproj

```diff
@@ -24,21 +32,29 @@
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <Import Project="$(WpfCppProps)" />
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(Architecture)'!='arm64'">
     <!-- 
       Target assembly will be a privatized copy of D3DCompiler, like  
       d3dcompiler_47_cor3.dll
     -->
     <TargetName>$(D3DCompilerDllBaseName)$(D3DCompilerVersion)$(WpfVersionSuffix)</TargetName>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Architecture)'=='arm64'">
+    <TargetName>$(D3DCompilerDllBaseName)$(D3DCompilerVersion)</TargetName>
```

Comment:
Do we need to set this if we are not shipping the component?

Reply:
@azchohfi: Do you have any objections to skipping this entire project for ARM64, or should it be left as-is?

Reply:
I'd prefer to leave this as-is for now, if there are no objections.